src/Cornelis/Types/Agda.hs

```diff
@@ -103,26 +84,10 @@ instance Show a =>
               ((.)
                  showSpace (showsPrec 11 $ toList b2_a1hOm))))
 
-data Interval' a = Interval { iStart, iEnd :: !(Pos' a) }
-  deriving (Data, Eq, Ord, Functor, Foldable, Traversable, Generic)
-
-instance Show a => Show (Interval' a) where
-  showsPrec n (Interval s e) =
-    showParen (n >= 11) $ showString "Interval " . showsPrec 11 s . showSpace . showsPrec 11 e
-
-type IntervalWithoutFile = Interval' AgdaOffset
-
 type SrcFile = Maybe AbsolutePath
 
-data Pos' a = Pos
-  { p_line :: !LineNumber
-  , p_col  :: !a
-  }
-  deriving (Eq, Ord, Show, Data, Functor, Foldable, Traversable, Generic)
-
-
 newtype AbsolutePath = AbsolutePath { textPath :: String }
-  deriving (Eq, Ord, Data)
```

Comment:
No more data?

Reply:
Fair. Presumably it's there so Agda can do syb stuff and just happened to get copied along.